recipes/pymirc/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pymirc" %}
+{% set version = "0.25" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymirc-{{ version }}.tar.gz
+  sha256: 0ec860399f222af6e4880a8c86f02a380a932058b572d8cccd3ceef7094e8a84
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.9
+    - setuptools-scm
+    - setuptools_scm_git_archive
+  run:
+    - matplotlib-base >=2.2.2
+    - nibabel >=3.0
+    - numba >=0.39
+    - numpy >=1.15
+    - pydicom >=1.1
+    - python >=3.6,<3.9
```

Does this actually not work with Python 3.9?
```suggestion
    - python >=3.6
```

---

Hi Chris,

(1) It also works with python 3.9, so python >=3.6 is fine.
(2) already changed to LGPL-3.0-only in a another commit.

Should I create another pull request?

---

1. You should push changes to this branch rather than making a new PR.
2. I don't see the `-only` on GitHub, perhaps you didn't push your changes?

---

- just pushed the a new 0.26 version on github and pypi where python <3.9 is removed and LGPL-3.0-or-later is updated
- also pushed the updated meta.yaml to this branch

One more question: I use grayskull to auto create the meta.yaml from the pypi package, but it does not pick up "or-later" suffix. Any idea why that is?

---

I'm not entirely sure and it's often annoyingly ambiguous but maybe due to the `-only` classifier being used in the PyPI classifiers:
https://github.com/gschramm/pymirc/blob/8a01c80ed1373aa45e785d3cd1c23fd5b188a405/setup.py#L18

The `-or-later` variant is: `License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)`.